en/docs/observe/mi-observe/working-with-monitoring-dashboard.md

```diff
@@ -36,17 +36,17 @@ You can use the dashboard to perform the following administration tasks related
 
 -   <b>View, update, and add loggers</b>
 
-    This page can be accessed by users with admin rights only. You can view log config information of each instance and update the log level. You can update the log levels on a single node or apply the change on entire cluster/group as well. Furthermore you can add new loggers, which will be applied to entire cluster/group.
+    This page can be accessed by users with admin rights only. You can view log configurations of each instance and update the log level. You can update the log levels on a single node or apply the change to the entire cluster/group as well. Furthermore, you can add new loggers, which will be applied to the entire cluster/group.
 
 -   <b>Manage users</b>
 
     This page can be accessed by users with admin rights only. You can view details of users stored in the [external user store]({{base_path}}/install-and-setup/setup/mi-setup/user_stores/managing_users). You can also add new users to the specified cluster/group.
 
-[comment]: <> (You can refer to the following video to get a quick understanding of how this is done.)
+## Using the MI Dashboard
 
-[comment]: <> (<iframe width="560" height="315" src="https://www.youtube.com/embed/WxcHkJVOgOU" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>)
+Follow the steps given below to get started with the Micro Integrator Dashboard.
 
-## Before you begin
+### Step 1 - Downloading the MI Dashboard
```

Comment:
As the steps all give the user a specific instruction, it should be worded as though an instruction is given. So "Downloading" should be "Download". Please fix this in the subsequent instructions as well.

Reply:
@Mariangela - The general recommendation for technical documentation is to use the 'ing' form (gerund form) in titles. However, to keep all docs consistency, will change this 'Download' as per your suggestion.